dist/stat_data.py

```diff
@@ -318,6 +318,9 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_queue_not_empty', 'eviction server candidate queue not empty when topping up'),
     CacheStat('cache_eviction_server_evicting', 'eviction server evicting pages'),
     CacheStat('cache_eviction_server_slept', 'eviction server slept, because we did not make progress with eviction'),
+    CacheStat('cache_eviction_skip_pages_retry', 'skip pages to avoid retry that have been tried to evict before'),
+    CacheStat('cache_eviction_skip_pages_last_running', 'skip pages that are written with transactions greater than the last running'),
+    CacheStat('cache_eviction_skip_queuing_dirty_pages_during_checkpoint', 'skip queuing dirty pages during checkpoint'),
```

Comment:
```suggestion
    CacheStat('cache_eviction_skip_queuing_dirty_pages_during_checkpoint', 'skip dirty pages during a running checkpoint'),
```

Reply:
Fixed in f6176b656d9a0112686d210cdf8d28783dc815e8